staging/src/kubevirt.io/api/usb/v1alpha1/types.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NodeConfig represents a subset of Node resources that we want to expose
+// to virtual machines.
+//
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
+// +genclient:nonNamespaced
+type NodeConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              NodeConfigSpec   `json:"spec" valid:"required"`
+	Status            NodeConfigStatus `json:"status,omitempty"`
+}
+
+type NodeConfigSpec struct {
+	USB USB `json:"usb,omitempty"`
+}
+
+// USB defines a group of USB devices based on its selectors and combine them under a single
+// resource name, to be allocated to the VM
+type USB struct {
+	// Identifies the list of USB host devices.
+	// e.g: kubevirt.io/storage, kubevirt.io/bootable-usb, etc
+	ResourceName string `json:"resourceName"`
+	// +listType=atomic
+	USBHostDevices []USBHostDevices `json:"usbHostDevices,omitempty"`
+}
+
+type USBHostDevices struct {
+	// The vendor:product of the devices we want to select.
+	// e.g: "0951:1666"
+	SelectByVendorProduct string `json:"selectByVendorProduct"`
+	// TODO SelectBySerialNumber: e.g: "E0D55E6CBD23E691583A004D"
+	// TODO SelectByBusDeviceNumber: e.g: "02-03"
+}
+
+// TODO
```

Comment:
TODO

Reply:
Some `TODO` were left on purpose to discuss it, like the other selectors above. This one is related to `type NodeConfigStatus struct{}` not being implemented, which I'm not sure if we want or not to implement it.

IIRC, if we could have multiple `USBDevicesConfig` CR, we don´t want to implement this. What @alicefr suggested in https://github.com/kubevirt/kubevirt/pull/10015#discussion_r1252693529 on having a single CR, then this would probably need to be implemented, it this correct @xpivarc? 

I'll reply on multiple-or-not CR in the other comment thread